go_utils/photo_download.py

```diff
@@ -354,6 +462,13 @@ def download_lc_photos(
         The column name of the column that contains the east photo urls. If not specified, these photos will not be included.
     west_photo : str, default = "lc_WestPhotoUrl"
         The column name of the column that contains the west photo urls. If not specified, these photos will not be included.
+    exclude_from_name : list of str, default=[]
+        A list of coloumn names to exclude from the downloaded photo names.
+        Accepted Excluded Names include: "direction", "latitude", "longitude", "date_str", "lc_id"
+    additional_name_stem : str, default=""
+        Additional custom information the user can add to the name.
+        NOTE: Name format is as follows: "lc_{additional_name_stem}_{direction}_{latitude}_{longitude}_{date_str}_{lc_id}_{photo_id}.png"
+            Excluded name fields will not appear in the name
```

Comment:
Could we (this would kind of be a major change) instead have like a format string set up so that the user could cherry-pick exactly which fields they'd like: e.g. `(f"{latitude}_{longitude}_{direction}", latitude="lc_Latitude")` and then have default arguments for those things?

Reply:
done! -- I changed the logic to instead ask for fields to INCLUDE in the name, in any order that the user would like. I also updated the flags + tests + documentation to reflect this.

However, I'm not sure allowing the flexibility of `latitude='lc_latitude'` is the best approach, since the column name was already specified in the `get_lc_download_targets` method. I think it's safe to assume the same column will contain latitude information.

Reply:
That's true, I think this current setup works!